scripts/install

```diff
@@ -31,6 +31,9 @@ ynh_script_progression "Adding $app's configuration..."
 mkdir -p $install_dir/smb.conf.d
 ynh_config_add --template="global-smb.conf" --destination="$install_dir/smb.conf.d/0-global.conf"
 ynh_config_add --template="share-smb.conf" --destination="$install_dir/smb.conf.d/share.conf"
+service_name=$(hostname)
+service_name=${service_name:-yunohost}
```

Comment:
Actually, that could be a `ynh_app_setting_set`, right? That could even be a `[install.discovery_name]` in `manifest.toml`? I'm sorry, this PR is a bit messy. This is my first contribution to YNH packaging.